index.mjs

```diff
@@ -79,6 +80,8 @@ export function $(pieces, ...args) {
 
 export const argv = minimist(process.argv.slice(2))
 
+export const globby = Object.assign(globbyModule.globby, globbyModule)
```

Comment:
This is little bit strange. Let’s export only globby, and if globbySync is needed user can import it as package will already there. 

what do you think?

Reply:
I see, but let’s globalise only promise based api and not modify package’s func.

Reply:
I think that to prevent mutation of `globbyModule.globby` it could be better to wrap it and leave the rest methods on it like `globby.globbySync` for consistency with the rest exposed packages.

Reply:
globby util methods are also commonly used. Injection is better than exposition each of them as a separate const, imo.

Reply:
Got it, thanks for fixes.